python/mxnet/contrib/onnx/mx2onnx/_op_translations.py

```diff
@@ -1666,3 +1666,26 @@ def convert_size(node, **kwargs):
     and return the created node.
     """
     return create_basic_op_node('Size', node, kwargs)
+
+
+@mx_op.register("log_softmax")
+def convert_logsoftmax(node, **kwargs):
+    """Map MXNet's log_softmax operator attributes to onnx's LogSoftMax operator
+    and return the created node.
+    """
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    # Converting to int
+    axis = int(attrs.get("axis", -1))
+    temp = attrs.get("temperature", 'None')
+    if temp != 'None':
+        raise AttributeError("ONNX supports only temperature=None")
```

Comment:
Can you include operator name in the message so we give complete information to the user that in logsoftmax, ONNX support only temperature=None but give - temp

Reply:
submitted the change. corrected this in error messages in other operators too